src/chrome/content/rules/Mono-project.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="Mono project">
+	<target host="www.mono-project.com" />
```

Comment:
```
mono-project.com
download.mono-project.com
origin-download.mono-project.com
news.mono-project.com
jenkins.mono-project.com
```

Reply:
https://mono-project.com and https://news.mono-project.com/ are not available.

Reply:
@franklinyu Can verify.